webapp/src/shared/components/Game/hooks/useOnlineGame.tsx

```diff
@@ -29,6 +29,7 @@ const SHOULD_CONFIGURE_GAME = 'shouldConfigureGame';
 const GET_SHOULD_CONFIGURE_GAME = 'getShouldConfigureGame';
 const GAME_CONFIG_SUBMIT = 'gameConfigSubmit';
 const MAX_CONNECTION_LAG_IN_MS = 100;
+const WARN_INTERVAL_IN_MS = 5000;
```

Comment:
I'd configure this as 10 seconds, 5 is still too low

Reply:
const usersColumns = Object.values(userKeys)
      .map(
        (column) => `users.${column} as user${capitalizeFirstLetter(column)}`,
      )
      .join(', ');